### DISCUSSION TITLE
Welcome to puzzle-tokens Discussions!

### DISCUSSION THREAD

**Discussion by MaxBazarov** [ingrammicro/puzzle-tokens#41] [Announcements]
Old discussions are available for some time here - 
https://spectrum.chat/puzzle-tokens?tab=posts

**Comment by bmooij-lg** [comment]
If I'd like to check up on https://spectrum.chat/puzzle-tokens/general/baseline-shift~79fbe23c-0440-4358-8f4a-b3cdd2efb0bc
Should I post about it here, or in the spectrum chat for now?
As a baseline function would be a potential fix for a long lasting sketch issue. (And boost my design system quite a bit!)
Thanks

**Reply by MaxBazarov** [reply]
@bmooij-lg  Please post here. Spectrum is no more a good place for discussions.
About baseline — I can not support it for text fragments, only for a whole text layer.
Will It  be Ok fo you?

**Reply by bmooij-lg** [reply]
I would like to apply this to font styles, so complete text layers indeed.
I'm generating many font styles with a certain font family which renders too low.
And the 'fix' in sketch is to manually move up the baseline multiple times (manually per step) per font style...

Adding the baseline when generating any font style with this font would probably omit a sketch bug, and add some great vertical spacing flexibility
